services/autorust/codegen/src/codegen.rs

```diff
@@ -130,6 +130,8 @@ pub fn parse_query_params(uri: &str) -> Result<HashSet<String>> {
 
 #[derive(Clone)]
 pub struct TypeNameCode {
+    /// Whether or not to pass a type as a reference.
+    is_ref: bool,
```

Comment:
The other bools in this struct are not prefaced with `is_`.  Consider `optional`, which similarly could be named `is_optional`.    What are your thoughts on naming this `reference` to follow the convention, and the related function `pub fn reference(..., reference: bool)`?